### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Zeph180**
Get an api to build an app around
Implement the home page interactions
Implement details page interactions
Create unit tests for pure functions in your Redux code.
Filter the data that you retrieve from the API using a Filter stateless component.
Host o netlify

### CODE REVIEW

File: src/components/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <Icon icon="material-symbols:settings" />
+        </div>
+      </div>
+      <section className="home-cont">
+        {cryptoCoins.map((cryptoCoin) => (
+          <article className="coin" key={cryptoCoin.id}>
+            <Link to={`touch-coin/details/${cryptoCoin.id}`}>
+              <Icon className="link" icon="mdi:arrow-right-thin-circle-outline" />
+            </Link>
+            <article className="logo-cont">
```
Comment (codepantha) [inline src/components/pages/Home.js]: - [x] Please implement a `Filter` stateless component as a way to filter the coins using given parameters e.g coin name as per requirement 👍 
image

File: src/App.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.App {
+/* .App {
```
Comment (codepantha) [inline src/App.css]: - [x] Please delete the `App.css` file since its content are commented and, therefore, not in use 👍

File: src/index.css [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  padding: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
```
Comment (codepantha) [inline src/index.css]: - [x] Please use `Lato` font face as specified in the Behance design Typography guideline

File: src/components/pages/Home.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (codepantha) [inline src/components/pages/Home.js]: - [ ] Please do not disable linters this way. Linters help you keep your code clean, tidy, uniform and of high quality 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Zeph180 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [x] A particular test that appears to be part of the boilerplate code in `App.test.js` is failing as can be seen below 👇. Please remove it.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Zeph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**